src/object_internal.h

```diff
@@ -340,7 +340,7 @@ DISPATCH_ENUM(dispatch_invoke_flags, uint32_t,
 #define _DISPATCH_INVOKE_AUTORELEASE_MASK	  0x00300000u
 );
 
-enum {
+enum : unsigned long {
```

Comment:
this syntax is not supported by C mode on clang I think, can you use DISPATCH_ENUM() instead?

Reply:
Done